### PR TITLE
configuration

### LINKED ISSUE #34: configure healthcheckbot
this bot should "check" if these services are up and running
- https://api-opis.unictdev.org/
- https://telegram.unictdev.org/
- https://open-job-dmi.unictdev.org/

### PR DESCRIPTION
**Pull request by Picred**
- Ho impostato gli url da verificare;
- Il controllo lo fa ogni 3600 secondi (1 ora);
- Ho creato uno script `run.sh` che avvia il `src/main.py`;
- Ho aggiornato il README con le dovute istruzioni per l'avvio;
- Lo script si può avviare in background con il comando dovuto e si può fermare creando un file `stop.flag`;
- Il file `settings.yaml` l'ho lasciato così (senza la versione `.dist`) perchè non penso ci siano _dati sensibili_.
- Mi servirebbe una mano per i **test** se dobbiamo considerarli (_fallivano anche prima_) 

closes #34

### CODE REVIEW

File: config/settings.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+run_timeout: 3600
```
Comment (Helias) [inline config/settings.yaml]: this should not be included in the PR, this file should be in .gitignore

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    if [ -f "$stop_flag" ]; then
+        cleanup
+    fi
+
+    if ! python3 src/main.py; then
+        echo "Error in src/main.py..."
+    fi
+
+    sleep "$run_timeout"
+done
```
Comment (Helias) [inline run.sh]: what is this?

File: src/urls.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+    "get": [
+        "https://api-opis.unictdev.org/", 
+        "https://telegram.unictdev.org/",
+        "https://open-job-dmi.unictdev.org/"
+    ], 
+    "ping": [
+        "https://api-opis.unictdev.org/", 
+        "https://telegram.unictdev.org/",
+        "https://open-job-dmi.unictdev.org/"
+    ]
```
Comment (Helias) [inline src/urls.json:11]: mhm, ideally we should move this in the settings yaml conf file, to keep the software "generic and configurable", but we could do this in another PR

File: config/settings.yaml (hunk 1 above)
Comment (Picred) [reply config/settings.yaml]: Do you mean the "settings.yaml" file?

File: run.sh (hunk 2 above)
Comment (Picred) [reply run.sh]: The script will start `src/main.py` every hour to avoiding continuous requests. I was thinking that doing this all the time might be resource consuming

File: config/settings.yaml (hunk 1 above)
Comment (Helias) [reply config/settings.yaml]: yes

File: run.sh (hunk 2 above)
Comment (Helias) [reply run.sh]: we could do this using cronjob, check what is cron and how it works
here is an example of cronjob https://github.com/azerothcore/telegram-automated-db-backup#optional-run-it-daily-using-crontab to run a script everyday

linux and mac supports it easily

So this script is not necessary but you could keep it if you want

File: README.md [hunk 4]
```diff
@@ -20,6 +20,16 @@ export QDBotIDs="10000000"                                      # Single ID
...
 
+### Run it every 5 minutes using crontab
+- Open crontab using:
+```bash
+$ crontab -e
+```
+- Add the following line
+```bash
+*/5 * * * * cd /path/to/Healthcheck-bot/ && python3 src/main.py > checks.log 2> errors.log    
+```
```
Comment (Helias) [inline README.md:31]: good 👍

File: src/urls.json [hunk 5]
```diff
@@ -1,4 +1,12 @@
 {
```
Comment (Helias) [inline src/urls.json:1]: the settings.yaml.dist is empty, you could put a placeholder with
`run_timeout: 3600`

**Comment by Helias** [comment]
something is failing in the tests

File: src/urls.json (hunk 5 above)
Comment (Picred) [reply src/urls.json:1]: But now `run_timeout` is no more used so I deleted it

**Review by Helias — APPROVED** [review]
